src/coreclr/src/jit/jitconfigvalues.h

```diff
@@ -394,6 +394,11 @@ CONFIG_INTEGER(JitGuardedDevirtualizationGuessUniqueInterface, W("JitGuardedDevi
 CONFIG_INTEGER(JitGuardedDevirtualizationGuessBestClass, W("JitGuardedDevirtualizationGuessBestClass"), 1)
 #endif // DEBUG
 
+// Enable insertion of patchpoints into Tier0 methods with loops.
+CONFIG_INTEGER(JitPatchpoint, W("JitPatchpoint"), 0)
```

Comment:
Do you enable OSR by setting `COMPlus_JitPatchpoint=1`? Seems like "OSR" and/or "On Stack Replacement" wording should be here, and maybe make it clear that this is the "root of all enablement", if so.

Reply:
Good point, I should rename this to `COMPlus_TC_OnStackReplacement` or similar.

You also need to set `COMPlus_TC_QuickJitForLoops=1` to turn this on.

Reply:
> You also need to set COMPlus_TC_QuickJitForLoops=1 to turn this on.

Could (should) you make `COMPlus_TC_OnStackReplacement` automatically set `COMPlus_TC_QuickJitForLoops=1`?

Reply:
You could at least document that requirement here.